includes/class-pureclarity-cron.php

```diff
@@ -97,9 +97,13 @@ private function create_schedule() {
 	 * Runs outstanding delta tasks
 	 */
 	public function run_delta_schedule() {
-		$this->process_products();
-		$this->process_categories();
-		$this->process_users();
+		if ( false === $this->settings->is_delta_running() ) {
+			$this->settings->set_is_delta_running( '1' );
+			$this->process_products();
```

Comment:
try/catch, so we always set the delta flag back?

Reply:
each of the child methods have their own try - catches which encapsulate all code in them, so in theory no exceptions should make it up to this function. I'll look at improving the category function though, as this has a couple of bits outside of it's try catch which could.